go.mod

```diff
@@ -7,8 +7,8 @@ require (
 	github.com/google/go-cmp v0.5.5
 	github.com/onsi/ginkgo v1.14.0
 	github.com/onsi/gomega v1.10.1
-	github.com/prometheus/common v0.31.0
-	github.com/prometheus/prometheus v2.5.0+incompatible
+	github.com/prometheus/common v0.32.1
```

Comment:
Could you partially pick this change from https://github.com/kubernetes-sigs/metrics-server/commit/6f2c2cabd4c2b643e73f801674a5f98e62d55f83? That would prevent the branch from diverging.